src/app/contact/contact-form/contact-form.component.scss

```diff
@@ -46,8 +46,8 @@
     			border-radius: 4px;
     			vertical-align: middle;
     			box-sizing: border-box;
-    			*display: inline;
-    			*zoom: 1;
+    			display: inline;
+    			zoom: 1;
```

Comment:
It was already there, I just removed the asteriks because it was giving error when I ran the tests locally.